tests/nodeunit/unit/views/contacts_by_freetext.js

```diff
@@ -0,0 +1,100 @@
+const utils = require('./utils');
+
+const doc = {
+  _id: '3c0c4575468bc9b7ce066a279b022e8e',
+  _rev: '2-5fb6ead9b03232a4cf1e0171c5434469',
+  name: 'Test Contact of Clinic',
+  date_of_birth: '',
+  phone: '+13125551212',
+  alternate_phone: '',
+  notes: '',
+  type: 'person',
+  reported_date: 1491910934051,
+  transitions: {
+    maintain_info_document: {
+      last_rev: 2,
+      seq: '241-g1AAAACbeJzLYWBgYMpgTmEQTM4vTc5ISXLIyU9OzMnILy7JAUklMiTV____PyuDOdE1FyjAnpRikZxsaYpNAx5j8liAJEMDkPoPNW0N2LQ0S_PkNNM0bPqyAMhOMjM',
+      ok: true
+    }
+  }
+};
+
+const nonAsciiDoc = {
+  _id: '3e32235b-7111-4a69-a0a1-b3094f257891',
+  _rev: '1-e19cb2355b26c5f71abd1cc67b4b1bc0',
+  name: 'बुद्ध Élève',
+  date_of_birth: '',
+  phone: '+254777444333',
+  alternate_phone: '',
+  notes: '',
+  parent: {
+    _id: 'd978f02c-093b-4266-81cd-3983749f9c99'
+  },
+  type: 'person',
+  reported_date: 1496068842996
+};
+
+const flatten = (array) =>
```

Comment:
Move this into utils? It's only used here, but could be used elsewhere. Or just include underscore? It's a test so bloat doesn't really matter...